rpcs3/Emu/RSX/GL/GLOverlays.cpp

```diff
@@ -469,28 +469,57 @@ namespace gl
 			"\n"
 			"uniform float gamma;\n"
 			"uniform int limit_range;\n"
-			"uniform int stereo;\n"
+			"uniform int stereo_display_mode;\n"
 			"uniform int stereo_image_count;\n"
 			"\n"
 			"vec4 read_source()\n"
 			"{\n"
-			"	if (stereo == 0) return texture(fs0, tc0);\n"
+			"	if (stereo_display_mode == 0) return texture(fs0, tc0);\n"
 			"\n"
 			"	vec4 left, right;\n"
 			"	if (stereo_image_count == 2)\n"
 			"	{\n"
-			"		left = texture(fs0, tc0);\n"
-			"		right = texture(fs1, tc0);\n"
+			"		vec2 coord_left = tc0 * vec2(1.f, 0.4898f);\n"
+			"		vec2 coord_right = coord_left + vec2(0.f, 0.510204f);\n"
+			"		if(stereo_display_mode == 1)\n" // anaglyph
```

Comment:
You can just format the enum values to a string representation in these comparisons.
There's no need to hardcode 1,2,3 etc.